backend/src/share/dto/myShareSecurity.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsBoolean, IsNumber, IsOptional } from "class-validator";
+
+export class MyShareSecurityDTO {
+	@IsBoolean()
+	passwordProtected: boolean;
+
+	@IsNumber()
+	@IsOptional()
+	maxViews: number;
```

Comment:
```suggestion
	@Expose()
	passwordProtected: boolean;

        @Expose()
	maxViews: number;
```

The validations aren't necessary as this DTO is only used for returning data.